packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -120,19 +132,25 @@ export class ChainService implements ChainServiceInterface {
     return obs;
   }
 
-  private async sendTransaction(transactionRequest: providers.TransactionRequest) {
+  private async sendTransaction(
+    transactionRequest: providers.TransactionRequest
+  ): Promise<providers.TransactionResponse | undefined> {
```

Comment:
Why should this function return a Promise<undefined>?

Reply:
It already does; it was just missing the return type (which is odd that the linter didn't notice?)